local/containers.go

```diff
@@ -144,7 +145,10 @@ func (cs *containerService) Run(ctx context.Context, r containers.ContainerConfi
 }
 
 func (cs *containerService) create(ctx context.Context, containerConfig *container.Config, hostConfig *container.HostConfig, networkingConfig *network.NetworkingConfig, name string) (string, error) {
-	created, err := cs.apiClient.ContainerCreate(ctx, containerConfig, hostConfig, networkingConfig, name)
+	created, err := cs.apiClient.ContainerCreate(ctx, containerConfig, hostConfig, networkingConfig, &v1.Platform{
```

Comment:
This cause error `1.41 is too new. Maximum supported API version is 1.40` on CI and on my Linux box :'(